packages/api/src/microservices/Assessment-Service/index.js

```diff
@@ -0,0 +1,9 @@
+const AssessmentService = require(`./Assessment-Service`);
+const UserService = require(`./User-Service`);
+const express = require('express');
+const { AssessmentService } = require('../microservices/Assessment-Service');
+
+module.exports = {
+  AssessmentService,
+  UserService,
+};
```

Comment:
When looking at the code on your computer, there was an error importing AssessmentService, so I would figure out the issue there